Automattic/jetpack
# Component: Adjust Pricing Slider thumb styling

Fixes #31478 

## Proposed changes:

* Adjust component pricing slider border and box-shadow styling according to the design prototype.

https://github.com/Automattic/jetpack/assets/6869813/a4891b31-d789-43bf-bef5-f0d694959f3f

### Other information:


## Jetpack product discussion



p1HpG7-ngN-p2

## Does this pull request change what data or activity we track or use?



N/A

## Testing instructions:






* Run the local storybook with `cd projects/js-packages/storybook && npm run storybook:dev`.
* Ensure the `Pricing Slider` thumb has a green border when the current value is not 0.
* Ensure the `Pricing Slider` thumb has the box shadow when focused.
